internal/cluster/statemachine.go

```diff
@@ -211,7 +260,10 @@ func NewClusterStateMachine(th *transitionHandler) stateswitch.StateMachine {
 		SourceStates: []stateswitch.State{
 			stateswitch.State(models.ClusterStatusInstalling),
 		},
-		Condition:        stateswitch.And(stateswitch.Not(th.IsFinalizing), stateswitch.Not(th.IsInstalling)),
+		Condition: stateswitch.And(
+			stateswitch.Not(th.IsFinalizing),
+			stateswitch.Not(th.IsInstalling),
+			stateswitch.Not(th.IsInstallingPendingUserAction)),
```

Comment:
Not sure in which case `stateswitch.Not(th.IsInstallingPendingUserAction)` is needed, i think that for this transition is `stateswitch.And(stateswitch.Not(th.IsFinalizing), stateswitch.Not(th.IsInstalling))` covering all the cases 
if you are not finilazing and not installing then  it means that one of the masters in error, it's not relevant if any of the hosts is pending for user action or not. 
Unless i missed something

Reply:
you are right, I missed that IsInstalling includes HostStatusInstallingPendingUserAction